test/Jsr292/src/com/ibm/j9/jsr292/indyn/IndyTest.java

```diff
@@ -333,28 +333,15 @@ public void test_indyn_ldc_BootstrapMethod_type() {
 		}
 	}
 	
-	//Negative test : Creating an invalid MethodType using LDC
+	//Negative test : Creating a MethodType with a non-existent class using LDC
 	@Test(groups = { "level.extended" })
-	public void test_indyn_ldc_invalid_type() {
-		
-		boolean typeNotPresentExceptionThrown = false;
-		
+	public void test_indyn_ldc_non_existent_class_methodtype() {
 		try {
 			MethodType mtToString = com.ibm.j9.jsr292.indyn.GenIndyn.ldc_invalid_type();
```

Comment:
`mtToString` is not used, can it be removed?

Reply:
we should keep it since it tells what is being returned.